src/card.tsx

```diff
@@ -626,89 +816,108 @@ const DisplayCard = React.forwardRef<HTMLDivElement, GenericDisplayCardProps>(
                     width="100%"
                     minHeight="100%"
                     isInverse={isInverse}
-                    background={isInverse && backgroundImage ? vars.colors.backgroundContainer : undefined}
+                    background={
+                        // Check for dark mode is needed until brandHighOnInverse token is created
+                        // https://github.com/Telefonica/mistica-design/issues/1215
+                        backgroundImage || backgroundVideo
+                            ? !isDarkMode && isExternalInverse
+                                ? vars.colors.brandHigh
+                                : vars.colors.backgroundContainer
+                            : undefined
+                    }
                 >
-                    <div
-                        className={styles.displayCard}
-                        style={{
-                            backgroundImage: backgroundImage
-                                ? `url("${CSS.escape(backgroundImage)}")`
-                                : undefined,
-                            paddingTop: withGradient && !icon && !hasTopActions ? 0 : 24,
-                        }}
-                    >
-                        {icon ? (
-                            <Box paddingBottom={withGradient ? 0 : 40} paddingX={24}>
-                                {icon}
-                            </Box>
-                        ) : (
-                            <Box paddingBottom={actions?.length || onClose ? (withGradient ? 24 : 64) : 0} />
-                        )}
-                        <Box
-                            paddingX={24}
-                            paddingTop={withGradient ? 40 : 0}
-                            paddingBottom={24}
-                            className={withGradient ? styles.displayCardGradient : undefined}
+                    <div className={styles.displayCardContainer}>
+                        <ThemeVariant isInverse={isExternalInverse}>
```

Comment:
why? :/

Reply:
I still dont get it. Shouldn't the card have a background? if the card has the corresponding backgorund, the outside background shouldn't affect in any case

Reply:
discussed offline. Resolving comment